### PR TITLE
docs: prerequisites section added

### LINKED ISSUE #40: Update readme instructions to include installing node
The readme needs to include a link on how to install node.
Without node, then you won't have access to `npm` and won't be able to use the Prettier format tool.

Just above the description in the readme add a `Prerequisites section` and list the link to install Node.js
https://docs.npmjs.com/downloading-and-installing-node-js-and-npm

### PR DESCRIPTION
**Pull request by AndreyDmitriev39r**
closes #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # CONTRAST RATIO APP
 
+## Prerequisites
+
+Install Node.js in order to have access to npm.
```
Comment (Okibe00) [inline README.md:5]: I think wrapping the link in a text will look better, I believe this is the markdown syntax `[text](url)`, you can convert the install into a link our create a new sentence like `click to install node.js .

Comment (AndreyDmitriev39r) [reply README.md:5]: Cool. You know you can actually request changes via GitHub interface? Remember I did that reviewing your PR?

**Review by Okibe00 — CHANGES_REQUESTED** [review]
please convert the install into a link or create a new sentence like `click to install node.js

**Comment by AndreyDmitriev39r** [comment]
@Okibe00 
I added some text and wrapped it into a link.
This link leads to the general instructions, and not hte installer directly, so my text is slightly different then one you suggested.
I will also request additional review from @sparkO7 to give her some practice

**Review by sparkO7 — APPROVED** [review]
LGTM!

**Pull request merged**